Installing Node.js Foundations Course : Fix wrong acronym (#27480)
TheOdinProject/curriculum: foundations/javascript_basics/installing_nodejs.md

Node.js is a JavaScript runtime environment that allows you to run JavaScript outside of your web browser. We will need this for some exercises in the upcoming lessons. To get started, there are some required tools we need before we can install Node on your system.

> We're going to install it using `nvm` (Node Version Manager), because it makes it easy to change Node versions and upgrade Node. There is another tool called `npm`. While `npm` is commonly understood to be an abbreviation for "Node Package Manager", it officially stands for "[npm Is Not An Acronym](https://twitter.com/npmjs/status/105690425242820608)". You will later use `npm` to install the various libraries and tools used in JavaScript environments. It can be easy to confuse these two, so read carefully!

Reviewer: Nit: removing "(Node Package Manager)" should be enough here, we don't need to get into what it stands for. 

```suggestion
We're going to install it using `nvm` (Node Version Manager), because it makes it easy to change Node versions and upgrade Node. There is another tool called `npm` that will use to install the various libraries and tools used in JavaScript environments. It can be easy to confuse these two, so read carefully!
```
